js/packages/ufg-client/src/client.ts

```diff
@@ -22,7 +23,17 @@ export function makeUFGClient({
   logger = makeLogger({logger, format: {label: 'ufg-client'}})
 
   const requests = makeUFGRequests({config, logger})
-  const fetchResource = makeFetchResource({fetchConcurrency: config.fetchConcurrency, logger})
+  const fetchResource =
+    process.env.APPLITOOLS_FETCH_RESOURCE_FROM_TUNNEL === 'true'
```

Comment:
I recommend this, but don't forget to import `utils`
```suggestion
    utils.general.getEnvValue('FETCH_RESOURCE_FROM_TUNNEL', 'boolean')
```